packages/2018-example-farmers-markets/src/components/FarmersMarketsOverTime/farmersMarketsOverTimeMeta.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Collapsable } from "@hackoregon/component-library";
+
+import FarmersMarketsOverTimeVisualization from "./FarmersMarketsOverTimeVisualization";
+
+const FarmersMarketsOverTimeMeta = (/* data */) => ({
+  title: "Farmers Markets Trending Upward",
+  slug: "farmers-markets-over-time",
+  introText: (
+    <p>
+      Farmers&apos; markets saw steady growth through the 1990s into the
+      mid-2000s. The recession correlates with abnormal growth in the total
+      number of Farmers&apos; Markets. The last two years have shown no growth.
+      Is this plateau expected to continue? What causes growth or decline in
+      Farmers&apos; Markets?
```

Comment:
One way to prevent having to escape all of the HTML characters and make the text more readable and approachable is like so:
`<p>{/*template string here*/}</p>`

Reply:
This could be an improvement to our template in making it much easier to copy/paste content. Or we could just have people ESLint fix all fixable problems and end up with a bunch of characters like these